src/www/diag_backup.php

```diff
@@ -521,6 +521,9 @@ function restore_config_section($section_name, $new_contents)
                         elseif ($field['type'] == 'password'):?>
 
                         <input name="<?=$fieldId;?>" type="password" value="<?=$field['value'];?>" />
+<?php
```

Comment:
if you send a separate pull request we can merge this directly

Reply:
#2616